catalystwan/models/configuration/feature_profile/sdwan/application_priority/__init__.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2024 Cisco Systems, Inc. and its affiliates
+
+
+from typing import Annotated, List, Union
```

Comment:
```suggestion
from typing_extensions import Annotated
```

this is for py3.8 to pass unit tests

Reply:
fixed